src/lib.rs

```diff
@@ -521,6 +568,10 @@ impl SMC {
         self.0.read_key(key)
     }
 
+    pub fn write_key<T: SMCType>(&self, key: FourCharCode, value: T) -> Result<(), SMCError> {
```

Comment:
Nope. Writing keys in smc is dangerous, don't let users shoot themselves in the foot and check the value before. Also the PR is about adding some power system.

Reply:
Got it. I was wondering why that function was missing. I see there is a reason now :-)